tests/integration_tests/functional/test_api.py

```diff
@@ -980,6 +982,104 @@ def test_api_balloon(test_microvm_with_ssh_and_balloon):
     assert test_microvm.api_session.is_status_bad_request(response.status_code)
 
 
+def test_get_full_config(test_microvm_with_ssh_and_balloon):
```

Comment:
We should also add a test to cover that PATCH reflects changes in the exported config.

Reply:
Right! Will do.

Reply:
done! 👍